tools/ratbagctl.test.in

```diff
@@ -317,17 +317,17 @@ class TestRatbagCtlResolution(TestRatbagCtl):
         command = "resolution active set"
         r = self.launch_good_test("test_device resolution active get")
         self.assertEqual(int(r), 2)
-        r = self.launch_good_test("test_device " + command + " 0")
+        r = self.launch_good_test("test_device " + command + " 1")
```

Comment:
Resolution 0 has a 0 dpi, and users shouldn't switch to such resolutions, that's why we need a change here (except if we remove the checks).

Reply:
First comment outdated, but we should add an `is_disabled` and `set_disabled`/`set_enabled` check.